rerun_cpp/src/rerun/half.hpp

```diff
@@ -0,0 +1,10 @@
+#pragma once
+
+#include <cstdint>
+
+namespace rerun {
+    /// IEEE 754 16-bit half-precision floating point number.
+    struct half {
```

Comment:
We can change this in the future - I just want a placeholder for now so that we don't have ugly holes in our C++ API